Sources/SwiftDocC/Semantics/Metadata/PageImage.swift

```diff
@@ -11,19 +11,26 @@
 import Foundation
 import Markdown
 
+/// A directive to associate an image with a page.
+///
+/// This directive can be used to control the image used both when rendering an icon representing the page such as
+/// in the navigator sidebar, and when rendering a card representing a page, e.g. in a  Topics section styled as a grid.
 public final class PageImage: Semantic, AutomaticDirectiveConvertible {
     public let originalMarkup: BlockDirective
     
+    /// The purpose for which to use this iamge.
     @DirectiveArgumentWrapped
     public private(set) var purpose: Purpose
     
+    /// The name and file extension of an image in your DocC catalog.
```

Comment:
Interesting... I see this is what we're using for the `@Image` docs but technically the file extension is optional and we refer to them as "documentation catalogs" not "DocC catalogs".

```suggestion
    /// The base file name of an image in your documentation catalog.
```